sdkconfig.defaults

```diff
@@ -7,14 +25,14 @@
 #
 # Serial flasher config
 #
-#CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
-#CONFIG_ESPTOOLPY_BAUD_115200B=y
+CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
+# CONFIG_ESPTOOLPY_BAUD_115200B is not set
 # CONFIG_ESPTOOLPY_BAUD_230400B is not set
-# CONFIG_ESPTOOLPY_BAUD_921600B is not set
+CONFIG_ESPTOOLPY_BAUD_921600B=y
```

Comment:
Faster? That doesn't make sense :package: 
Oh well, I don't think this setting is important since we kinda never use the C only fw anymore?